index.js

```diff
@@ -51,11 +51,16 @@ StreamProvider.prototype.isConnected = function(){
 StreamProvider.prototype._onResponse = function(response){
 
   const id = response.id
+  if (id === null || id === undefined) {
+    return
+  }
 
   const data = this._payloads[id]
-  if (!data) throw new Error(
-    `StreamProvider - Unknown response id for response: ${response}`
-  )
+  if (!data) {
+      throw new Error(
+      `StreamProvider - Unknown response id for response: ${response}`
+    )
+  }
```

Comment:
is this just a formatting preference or was something actually broken here?

Reply:
formatting preference. i can revert this and make the guard i added match the same syntax style if preferred

Reply:
No its alright, I think we prefer to always use brackets, I was just making sure I wasn't missing something obvious